nifi-toolkit/nifi-toolkit-cli/src/main/java/org/apache/nifi/toolkit/cli/impl/client/nifi/ProcessGroupClient.java

```diff
@@ -75,4 +75,6 @@ FlowEntity copySnippet(String processGroupId, CopySnippetRequestEntity copySnipp
         throws NiFiClientException, IOException;
 
     FlowComparisonEntity getLocalModifications(String processGroupId) throws NiFiClientException, IOException;
+
+    void deleteProcessGroup(ProcessGroupEntity entity) throws NiFiClientException, IOException;
```

Comment:
As other delete commands return the deleted entity, that looks like a good pattern to follow in this scenario.

Reply:
done